docs/content/latest/explore/ysql-language-features/partitions.md

```diff
@@ -21,25 +21,23 @@ Partitioning is another term for physically dividing large tables in YugabyteDB
 
 Since partitioned tables do not appear nor act differently from the original table, applications accessing the database are not always aware of the fact that partitioning has taken place.
 
-It is recommended to use partitioned tables with a query planner that can decide which tables to access and which ones to ignore based on the table content. 
-
 YSQL supports the following types of partitioning:
 
 - Range partitioning, when a table is partitioned into ranges defined by one or more key columns. In this case, the ranges of values assigned to partitions do not overlap.
 - List partitioning, when a table is partitioned via listing key values to appear in each partition.
 
 ## Declarative Table Partitioning
 
-YSQL allows you to specify how exactly to divide a table. You provide a partitioning method and partition key consiting of a list of columns or expressions. The devided table is called a partitioned table, and the resulting tables are called partitions. When you insert rows into a partitioned table, they are redirected to a partition depending on the value of the partition key.
+YSQL allows you to specify how exactly to divide a table. You provide a partitioning method and partition key consiting of a list of columns or expressions. The divided table is called a partitioned table, and the resulting tables are called partitions. When you insert rows into a partitioned table, they are redirected to a partition depending on the value of the partition key.
```

Comment:
Please add at the end of this paragraph: You can also directly insert rows into the partition table itself, and those rows can be fetched by querying the parent table.

Reply:
Added this sentence. Here's what we have now:
YSQL allows you to specify how exactly to divide a table. You provide a partitioning method and partition key consisting of a list of columns or expressions. The divided table is called a partitioned table, and the resulting tables are called partitions. When you insert rows into a partitioned table, they are redirected to a partition depending on the value of the partition key. You can also directly insert rows into the partition table itself, and those rows can be fetched by querying the parent table.